content/nginxaas-azure/app-protect/configure-waf.md

```diff
@@ -101,7 +101,33 @@ The following table shows the path to the precompiled policy file that needs to
 
 To view the contents of the available security policies, navigate to the azure portal and select the **Security Policies** tab in the App Protect section.
 
-{{<note>}}Custom policies are not supported at this time.{{</note>}}
+## Custom Policies
+
+NGINXaas for Azure also supports custom security policies. You can create and modify custom security policies to deploy to NGINX App Protect Instances using the API or Azure Portal.
+
+### Manage Custom Policies
+
+To create a custom security policy in the Azure Portal:
+
+1. Select your deployment
+2. Select **NGINX app protect WAF** from the menu on the left
+3. Select **Custom Policies**
+4. Select **Add Custom Security Policy** to open the policy editor
+
+In the policy editor, enter the **Name**, **File path**, your policy content, and then select **Save**. The **File path** is optional and will default to the path "/etc/app_protect/conf/" plus the policy **Name** with a ".json" extension. For more information on policy configuration and syntax, refer to the NGINX App Protect [configuration guide](https://docs.nginx.com/nginx-app-protect-waf/v5/configuration-guide/configuration/).
+
+   {{< img src="nginxaas-azure/custom-policy-editor.png" alt="NGINXaaS Custom Policy Editor." >}}
+
+After your policy has been saved, you can then reference it in your NGINX configuration the same as you did for precompiled policies.
+
+{{<note>}}The **name** field within the security policy must be unique among the policies referenced in your NGINX configuration.{{</note>}}
+{{<note>}}Referencing both custom and precompiled policies in your NGINX configuration is not supported at this time. See the [Known issues]({{< ref "/nginxaas-azure/known-issues.md" >}}) for a workaround for this limitation.{{</note>}}
```

Comment:
```suggestion
{{<note>}}The **name** field within the security policy must be unique among the policies referenced in your NGINX configuration.{{</note>}}

{{<warning>}}NGINXaaS does not currently support using both custom and precompiled policies in your NGINX configuration. See the [Known issues]({{< ref "/nginxaas-azure/known-issues.md" >}}) for a workaround for this limitation.{{</warning>}}
```

Reply:
also,

The workaround is a single line:
`Make a copy of the default policy you want to use, then add it as a custom policy with a different name.`

Try to avoid making users jump around pages for simple instructions

```suggestion
{{<note>}}The **name** field within the security policy must be unique among the policies referenced in your NGINX configuration.{{</note>}}

{{<warning>}}Referencing both custom and precompiled policies in your NGINX configuration is not supported at this time. 

As a workaround, make a copy of the default policy you want to use, then add it as a custom policy with a different name.
{{</warning>}}
```

Reply:
Fixed